compiler/visq/visqlib/QErrorComputer.py

```diff
@@ -24,6 +24,10 @@ class QErrorComputer:
     def __init__(self, fp32_dir, fq_dir):
         self._fp32_dir = fp32_dir
         self._fq_dir = fq_dir
+        self.qerror_map = dict()
+        self._num_processed_data = 0
+
+    def update_data(self, fp32_dir, fq_dir):
```

Comment:
Could you update the function name to `collect_data_path` ?

Reply:
Except that everything looks good to me.

Reply:
> Could you update the function name to `collect_data_path` ?

ok. done.